ios/Plugin/PrivacyScreenConfig.swift

```diff
@@ -1,5 +1,7 @@
 import Foundation
 
 public struct PrivacyScreenConfig {
-    var enable = true
+    var enable = true;
+    var useImageBackground = false;
```

Comment:
Please add the type definitions for this option to the `definitions.ts` file and run `npm run docgen`.

Reply:
It's done :)